conans/test/functional/generators/virtualenv_test.py

```diff
@@ -183,9 +183,12 @@ def _run_virtualenv(self, generator):
 
         env_before = _load_env_file(os.path.join(self.test_folder, self.env_before))
         env_after = _load_env_file(os.path.join(self.test_folder, self.env_after))
-        if platform.system() == "Darwin":
-            env_after.pop(six.u("PS1"), None)  # TODO: FIXME: Needed for the test to pass
-            env_after.pop("PS1", None)  # TODO: FIXME: Needed for the test to pass
+        # FIXME: Not the best behavior
+        # The deactivate sets PS1 always, but sometimes it didn't exist previously
+        there_was_ps1 = os.getenv("PS1")
```

Comment:
For some reason this is broken in CI: https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-7975/1/pipeline